src/features/profile/components/atoms/ProfilePicture.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from 'next/image';
+
+import { useAtomValue } from 'jotai';
+import { Pencil } from 'lucide-react';
+
+import { Button } from '@/components/ui/button';
+
+import { profileModeAtom } from '../../stores/profile-atoms';
+
+interface ProfilePictureProps {
+  src: string;
+}
+
+const ImageContainer = ({ children }: { children: React.ReactNode }) => (
+  <div className="relative h-[128px] w-[128px] overflow-hidden rounded-[64px] border border-[#e2e8f0]">
+    {children}
+  </div>
+);
+
+const StyledImage = ({ src }: { src: string }) => (
+  <Image
+    src={src}
+    alt="profile picture"
+    className="h-full w-full object-cover"
+    width={100}
+    height={100}
+  />
```

Comment:
```suggestion
  <Image
    src={src}
    alt="profile picture"
    fill
    className="object-cover"
  />
```

<Image /> 똑바로 사용해주세요